pos_partner_sale_warning/static/src/js/PartnerListScreen.esm.js

```diff
@@ -14,8 +15,8 @@ export const PartnerListScreenWarning = (PartnerListScreen) =>
             }
             if (partner && partner.sale_warn && partner.sale_warn !== "no-message") {
                 this.showPopup("ErrorPopup", {
-                    title: this.env._t(`Warning for ${partner.name}`),
-                    body: this.env._t(partner.sale_warn_msg),
+                    title: sprintf(this.env._t("Warning for %s"), partner.name),
```

Comment:
No need to use sprintf. Check following example
https://github.com/odoo/odoo/blob/457561aaa0fc6569a6c48d852b2b9bc0e5a1065f/addons/auth_password_policy/static/src/password_policy.js#L29

Reply:
@yelizariev  I think no. Because in 16 version used this: 
https://github.com/odoo/odoo/blob/a591e9c061374bfaf84d41a94246cff83ab1899c/addons/auth_password_policy/static/src/password_policy.js#L30

Reply:
Indeed, it's v17+ feature only https://github.com/odoo/odoo/pull/123932